security.rst

```diff
@@ -421,20 +423,22 @@ To fix this, add an ``encoders`` key:
     .. code-block:: php
 
         // app/config/security.php
+        use Symfony\Component\Security\Core\User\User;
+
         $container->loadFromExtension('security', [
             // ...
 
             'encoders' => [
-                'Symfony\Component\Security\Core\User\User' => 'plaintext',
+                User::class => 'plaintext',
             ],
             // ...
         ]);
 
-User providers load user information and put it into a ``User`` object. If
-you :doc:`load users from the database </security/entity_provider>`
+User providers load user information and put it into a :class:`Symfony\Component\Security\Core\User\UserInterface`
+implementation. If you :doc:`load users from the database </security/entity_provider>`
 or :doc:`some other source </security/custom_provider>`, you'll
-use your own custom User class. But when you use the "in memory" provider,
-it gives you a ``Symfony\Component\Security\Core\User\User`` object.
+use your own custom User class. But when you use the "in memory" provider type,
+it gives you a :class:`Symfony\Component\Security\Core\User\User` object.
```

Comment:
Minor issue here: when using `:class:` you must double all slashes.

Reply:
Fixed, thanks!